res/shaders/gba-color.shader/gba-color.fs

```diff
@@ -0,0 +1,42 @@
+varying vec2 texCoord;
+uniform sampler2D tex;
+uniform vec2 texSize;
+
+uniform float darken_screen;
+uniform float target_gamma;
+uniform float display_gamma;
+uniform float sat;
+uniform float lum;
+uniform float contrast;
+uniform float blr;
```

Comment:
This is a lot of uniforms. Could it be turned into a matrix, or a series of vec3s, seeing as it's composited into a matrix anyway?